graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -18,8 +18,11 @@
 class GdsArrowClient:
     @staticmethod
     def is_arrow_enabled(query_runner: QueryRunner) -> bool:
-        arrow_info = query_runner.call_procedure(endpoint="gds.debug.arrow", custom_error=False).squeeze().to_dict()
-        return not not arrow_info["running"]
+        arrow_server_running = (
+            query_runner.call_procedure(endpoint="gds.debug.arrow", custom_error=False, yields=["running"])
+            .squeeze()
+        )
+        return not not arrow_server_running
```

Comment:
Is this the typed python equivalent to the `!!foo` pattern in JS to turn a truthy value into True? 🙈

Reply:
I guess so, but I think that is not needed anymore now that only a single value is returned